packages/mdc-drawer/README.md

```diff
@@ -319,6 +319,7 @@ Mixin | Description
 `mdc-drawer-item-corner-radius($radius)` | Sets the corner border radius of the drawer list item.
 `mdc-drawer-activated-overlay-color($color)` | Sets the overlay color of the activated drawer list item.
 `mdc-drawer-scrim-fill-color($color)` | Sets the fill color of `mdc-drawer-scrim`.
+`mdc-drawer-z-index($value)` | Sets the z index of drawer. Drawer stays on top of top app bar except for clipped variant of drawer.
```

Comment:
This seems like it should be in a separate feat(drawer) PR. Can we isolate just the mdc-drawer-modal.scss change for this PR?

Reply:
Sent PR #3453